### PR TITLE
chore: bump cdk to 2.68.0

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

Moves CDK to 2.68.0
Changes imports of appsync L2 constructs as they graduated to stable.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,7 +139,7 @@
     "parse-path": "^5.0.0",
     "parse-url": "^8.1.0",
     "moment-timezone": "^0.5.35",
-    "aws-cdk-lib": "~2.53.0",
+    "**/aws-cdk-lib": "~2.68.0",
```
Comment (sobolk) [inline package.json:142]: previous declaration didn't do much. see https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

**Review by phani-srikar — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -7875,21 +7864,21 @@ aws-appsync@^4.1.1, aws-appsync@^4.1.9:
     url "^0.11.0"
     uuid "3.x"
 
-aws-cdk-lib@~2.53.0:
-  version "2.53.0"
-  resolved "https://registry.npmjs.org/aws-cdk-lib/-/aws-cdk-lib-2.53.0.tgz#c36ecfb87ba4e7d1cdfe0c620161beda0ce4aa67"
-  integrity sha512-ADpJ9luJxzmOrP/GJ37nA1YtdNmsOsjE+NZnQOpB7YL0spUxZJGNVAselaFLSDZmIsC6d9mSnW81fj4SDP1gAw==
+aws-cdk-lib@~2.53.0, aws-cdk-lib@~2.68.0:
```
Comment (phani-srikar) [inline yarn.lock:7867]: we'll probably need another update to this file once the CLI packages like `category-custom` and `cloudformation-provider` get updated to this higher version. That is where this `~2.53.0` is coming from.

Comment (sobolk) [reply yarn.lock:7867]: Correct, but we want your tests run with uniform cdk.

File: packages/amplify-graphql-function-transformer/src/graphql-function-transformer.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {
 } from '@aws-amplify/graphql-transformer-core';
 import { TransformerContextProvider, TransformerSchemaVisitStepContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
-import { AuthorizationType } from '@aws-cdk/aws-appsync-alpha';
+import { AuthorizationType } from 'aws-cdk-lib/aws-appsync';
```
Comment (alharris-at) [inline packages/amplify-graphql-function-transformer/src/graphql-function-transformer.ts:11]: Nice!

**Review by alharris-at — APPROVED** [review]

**Pull request merged**